lib/ash_admin/pages/page_live.ex

```diff
@@ -21,11 +21,12 @@ defmodule AshAdmin.PageLive do
   def mount(
         _params,
         %{
-          "prefix" => prefix
+          "prefix" => _prefix
```

Comment:
we can just use the `prefix` variable here, no need for the `prefix = session["prefix"]` below.